features/22-email-verification/api.md

```diff
@@ -47,14 +73,14 @@ Flow:
 When signing up:
 
 ```javascript
+// email
 skygear.auth.signupWithEmail("johndoe@example.com", "password")
 .then((user) => {
   console.log(skygear.auth.verified); // false
 });
-```
 
-```javascript
-skygear.auth.signupWithPhone("johndoe@example.com", "password")
+// phone
+skygear.auth.signupWithPhone("+85221559299", "password")
```

Comment:
is password optional when signing up with phone Number?

Reply:
this will be covered later with #19

Reply:
Common use case mentioned by PM says that it is common for apps that login with phone number to have no password for logging in. Therefore, this API for signing up with phone number *and* password is going to be less commonly used. But the spec currently is grown to the point it is not likely we will finish this feature in the upcoming milestone if we add login with email/SMS into this spec.

I am happy to add this spec for login with email/SMS if we found time to implement this towards the end of the upcoming milestone.